nbconvert/nbconvertapp.py

```diff
@@ -51,7 +51,9 @@ def validate(self, obj, value):
 nbconvert_aliases.update(base_aliases)
 nbconvert_aliases.update({
     'to' : 'NbConvertApp.export_format',
-    'template' : 'TemplateExporter.template_file',
+    'template' : 'TemplateExporter.template_name',
+    'theme' : 'HTMLExporter.theme',
+    'template-file' : 'TemplateExporter.template_file',
```

Comment:
Do we want ```template-file``` to be a part of the CLI API? I would think not. I also think that the ```theme``` alias isn't necessary and could be confused for having functionality with other exporters.

Reply:
Since the theme argument isn't working, i'm not even gonna argue :)

I think the template-file could be useful, as demonstrated in the examples below (with the latex template).